app/views/admin/users/_header.html.erb

```diff
@@ -0,0 +1,61 @@
+<% if @user.access_locked? %>
+  <div class="crayons-notice crayons-notice--danger py-2 pr-2 mb-3 flex justify-between items-center">
+    <p class="flex-1 flex">
+      <%= crayons_icon_tag(:lock, title: "Access locked", class: "mr-2") %>
+      This user has locked access.
+    </p>
+    <%= link_to "Unlock access", unlock_access_admin_user_path(@user), method: :patch, class: "c-link c-link--block c-link--branded" %>
+  </div>
+<% end %>
+
+<section class="crayons-card flex gap-4 m:gap-6 mb-2 m:mb-3 p-3 s:p-4 m:p-7">
+  <img src="<%= @user.profile_image_url_for(length: 192) %>" width="96" height="96" alt="<%= @user.name %> profile picture" class="radius-full bg-base-10 shrink-0 hidden s:block">
+  <div class="grid gap-2 flex-1">
+    <div class="s:flex items-center">
+      <div class="flex flex-1 items-center">
+        <h1 class="crayons-title lh-tight">
+          <%= @user.name %>
+        </h1>
+        <%= render "status" %>
+      </div>
+      <%= render "actions" %>
+    </div>
+    <div class="fs-s mb-1 color-secondary">
+      @<%= @user.username %> <span class="opacity-50">&bull;</span> ID <%= @user.id %> <span class="opacity-50">&bull;</span>
+      <% if @user.registered? %>
+        Member since
+        <time datetime="<%= @user.created_at.strftime("%Y-%m-%dT%H:%M:%S%z") %>" title="<%= @user.created_at.strftime("%Y-%m-%d, %H:%M") %>">
+          <%= @user.created_at.strftime("%b %e '%y") %>
+        </time>
+      <% else %>
+        <span class="color-accent-danger fw-medium">User has not accepted their invitation yet.</span>
+      <% end %>
+    </div>
+    <div>
+      <ul class="flex flex-col s:flex-row gap-3 s:gap-6 flex-wrap">
+        <li>
+          <a href="mailto:<%= @user.email %>" class="c-link c-link--icon-left inline-block" target="_blank">
+            <%= crayons_icon_tag(:email, title: "Email", class: "c-link__icon") %>
+            <%= @user.email %>
+          </a>
```

Comment:
So here's something weird...

https://user-images.githubusercontent.com/108287/150530847-dc33d41e-8f4e-4738-b204-e3cd5a58a57a.mov

Its the newlines that are causing those spaces between label and the icon and I don't know how to trim/strip these... Is ruby on rails really that silly? :D 

It obviously happens with all these links here... 🤷

Reply:
Just a note here: We'll look at this in a separate PR.